pkg/registry/generic/etcd/etcd_test.go

```diff
@@ -66,17 +67,22 @@ func hasCreated(t *testing.T, pod *api.Pod) func(runtime.Object) bool {
 func NewTestGenericEtcdRegistry(t *testing.T) (*tools.FakeEtcdClient, *Etcd) {
 	f := tools.NewFakeEtcdClient(t)
 	f.TestIndex = true
-	h := tools.NewEtcdHelper(f, testapi.Codec())
+	h := tools.NewEtcdHelper(f, testapi.Codec(), os.Getenv("ETCD_PREFIX"))
 	strategy := &testRESTStrategy{api.Scheme, api.SimpleNameGenerator, true, false}
+	podPrefix := "pods"
```

Comment:
or just eliminate, it's only needed two places and a string is just as easy to understand.